test/suites/zombie_tanssi_relay_eth_bridge/test_zombie_tanssi_relay_eth_bridge.ts

```diff
@@ -44,17 +45,19 @@ describeSuite({
             const relayNetwork = relayApi.consts.system.version.specName.toString();
             expect(relayNetwork, "Relay API incorrect").to.contain("dancelight");
 
+            relayCharlieApi = context.polkadotJs("Tanssi-charlie");
+
             // //BeaconRelay
             const keyring = new Keyring({ type: "sr25519" });
             alice = keyring.addFromUri("//Alice", { name: "Alice default" });
             const beaconRelay = keyring.addFromUri("//BeaconRelay", { name: "Beacon relay default" });
             const executionRelay = keyring.addFromUri("//ExecutionRelay", { name: "Execution relay default" });
 
             // Operator keys
-            operatorAccount = keyring.addFromUri("//" + "Bob", { name: "COLLATOR" + " ACCOUNT" });
-            operatorNimbusKey = keyring.addFromUri("//" + "COLLATOR_NIMBUS", { name: "COLLATOR" + " NIMBUS" });
-
-            await relayApi.tx.session.setKeys(u8aToHex(operatorNimbusKey), []).signAndSend(operatorAccount);
+            operatorAccount = keyring.addFromUri("//Charlie", { name: "Charlie default" });
+            operatorNimbusKey = await relayCharlieApi.rpc.author.rotateKeys();
```

Comment:
```suggestion
            // We rotate the keys for charlie so that we have access to them from this test as well as the node
            operatorNimbusKey = await relayCharlieApi.rpc.author.rotateKeys();
```